truss/contexts/image_builder/util.py

```diff
@@ -11,7 +11,7 @@
 # [IMPORTANT] Make sure all images for this version are published to dockerhub
 # before change to this value lands. This value is used to look for base images
 # when building docker image for a truss.
-TRUSS_BASE_IMAGE_VERSION_TAG = "v0.4.9"
+TRUSS_BASE_IMAGE_VERSION_TAG = "v0.4.9rc4"
```

Comment:
This seems ahead of the pyproject.toml version. We do rc's for a release and then the release. Going from 0.4.9 to 0.4.9rc seems backward.